index.html

```diff
@@ -2490,6 +2490,72 @@ <h3>
           </p>
         </aside>
       </section>
+      <section class="informative">
+        <h3>
+          Web Apps with scope conflicts
+        </h3>
+        <p>
+          Because scopes are based on URL matching, it is possible for a
+          developer to create multiple web applications with the same,
+          overlapping, or nested scopes. Doing so creates several issues
+          (detailed below) and is thus considered bad practice.
+        </p>
+        <p>
+          One origin per application is ideal because it isolates common user
+          agent settings and controls to apply to only one web applications.
+          Without this isolation, these settings or controls can apply to more
+          than one web application, which will likely be unexpected by the
+          user. Examples:
+        </p>
+        <ul>
+          <li>Permissions
+          </li>
+          <li>Storage and storage quota
+          </li>
+          <li>User settings (e.g. font size, zoom level)
+          </li>
+        </ul>
+        <p>
+          To aide with disambiguation apps with scope conflicts, user agents
+          can determine the <dfn>controlling app</dfn> for a given document
```

Comment:
+1 this feels like getting into normative (optional) requirements in a non-normative section. Look for instances of the word "can" and "may" in the above paragraphs and ask yourself whether you're specifying things for the user agent - I think you are, so those words should be capitalized ("can" should become "MAY") and moved into a normative section, or this section can become normative with notes for non-normative bits.

Conversely, the "may" in "The following scenarios may require" below is not a normative requirement so should be reworded to avoid the word "may" - e.g. "The following scenarios could require". Or turn it into a proper normative statement as Marcos is suggesting.